### PR TITLE
Benji/jib 207 error view

### PR DESCRIPTION
**Pull request by Benjamindodgson**

### CODE REVIEW

File: Shared/API/Onboarding+Cloud.swift [hunk 1]
```diff
@@ -37,7 +37,7 @@ struct SendCode: CloudFunction {
 }
 
 struct VerifyCode: CloudFunction {
-    typealias ReturnType = String
+    typealias ReturnType = [String: String]
```
Comment (Benjamindodgson) [inline Shared/API/Onboarding+Cloud.swift:40]: This now returns the sessionToken and a channelId, if one exists.

File: Shared/Flows/Onboarding/OnboardingViewController.swift [hunk 2]
```diff
@@ -72,13 +69,14 @@ class OnboardingViewController: SwitchableContentViewController<OnboardingConten
...
-        self.view.insertSubview(self.confettiView, aboveSubview: self.blurView)
-
         self.welcomeVC.$state.mainSink { (state) in
             switch state {
             case .welcome:
                 self.updateUI()
-            case .signup:
+            case .login:
+                Task {
+                    try await self.updateInvitor(with: "IQgIBSPHpE")
```
Comment (MartinJibber) [inline Shared/Flows/Onboarding/OnboardingViewController.swift:78]: What's this magic string?

File: Shared/Flows/Toast/ToastBannerView.swift [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+        self.descriptionLabel.alpha = 0
+        self.titleLabel.alpha = 0
+
+        self.descriptionText = localized(self.toast.description)
+        self.title = self.toast.title
+        self.imageView.displayable = self.toast.displayable
+    }
+
+    override func didReveal() {
+        self.moveLeft()
```
Comment (MartinJibber) [inline Shared/Flows/Toast/ToastBannerView.swift:63]: Should this call super? Just checking.

**Review by MartinJibber — APPROVED** [review]
Couple questions, but looks good.

**Pull request merged**